db/model/room.js

```diff
@@ -33,6 +34,14 @@ module.exports = function room(seq, dataTypes) {
       },
       comment: 'Create a named room ',
     },
+    settings: {
+      type: dataTypes.ARRAY(dataTypes.JSONB),
+      allowNull: true,
+      validate: {
+        contains: u.validateSettingsArray,
+      },
+      comment: 'Key/Value pairs for user specific settings',
```

Comment:
https://github.com/salesforce/refocus/blob/master/db/model/roomType.js#L49
We have already implemented as part of a past pr for room types so this would be keeping it consistent. If we are too change it on this pr we would then need a migration to change it for room types.